README.md

```diff
@@ -22,9 +28,76 @@ Code quality checking and enforcement is done with the following tools:
 
 Error and exception logging is done using Sentry.
 
-## Deployment
- - Provide `SENTRY_DSN` and `SENTRY_ENVIRONMENT` as environmental variables
-   during deployment.
+#### Environmental Variables
+
+| Name                                  | Description                                                                    | Default   |
+|---------------------------------------|--------------------------------------------------------------------------------|-----------|
+| DB_HOST                               | The MongoDB host to connect to.                                                | localhost |
+| DB_PORT                               | The port to connect to MongoDB on.                                             | 27017     |
+| DB_USER                               | The username to access the MongoDB instance.                                   | admin     |
+| DB_NAME                               | The name of the MongoDB instance.                                              | forms     |
+| DB_PASSWORD                           | The password to access the MongoDB instance.                                   | pwd       |
+| SENTRY_DSN                            | A Sentry error monitoring Data Source Name.                                    |           |
+| SENTRY_ENVIRONMENT                    | A Sentry error monitoring DSN environment.                                     | local     |
+| FEATURE_FORMR_PARTB_COVID_DECLARATION | this controls whether forms are stored in the S3 regardless of Lifecycle State | true      |
+| APPLICATION_FILESTORE_BUCKET          | the S3 bucket the forms are stored in.                                         |           |
+| APPLICATION_FILESTORE_ALWAYSSTORE     | this feature flag controls whether to show this section in FormR PartB         | false     |
+
+## Usage
+### Saving Forms
+
+To save the form that is received a key is made from the ObjectKeyTemplate, TraineeTisID and form
+Id and the metadata is mapped from the form into an ObjectMetadata and the submissionDate is generated
+in a LocalDateTime format. these objects are then made into a PutObjectRequest and put into the amazonS3
+bucket. 
+
+##### Save Forms Example
+```
+PUT api/forms/formr-parta
+```
+```
+PUT api/forms/formr-partb
+```
+
+### getFormRPartBsByTraineeTisId
+
+This method is used to return a collection of submitted and draft forms. when this request is 
```

Comment:
To be honest, I think this level of detail is going to make work for us down the line (as per comment above). It might be useful to think of writing these parts of the readme as being a bit similar to writing an end-to-end test: unless there is something really unexpected or sneaky in the code, the focus is just on what you put in and what you get out, and how it actually does it can be omitted.
So for this, I'd maybe just focus on the API, specifying how you make the call (with an authentication token that includes the trainee TIS id), and what you get back: a list of FormRPartSimpleDtos in JSON format (e.g. `[{"id":"62bac33d332e487b84a3bdcc","traineeTisId":"47165","submissionDate":"2022-06-28T09:01:00.337","lifecycleState":"SUBMITTED"}]`)

Reply:
If you think its useful, you could mention that all forms are returned, regardless of whether they are in draft or any other lifecycle state, but generally we can assume that the reader can figure out the implementation code, unless there's something particularly opaque - in which case that might really indicate an issue for us to address, in terms of simplifying the code.